CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+
+[Code of Conduct](#code-of-conduct)
+
+[License](#license)
+
+[How to report a bug?](#how-to-report-a-bug-🐛)
+
+[How to request the change?](#how-to-request-the-change-💡)
+
+[How to submit a change?](#how-to-submit-a-change-🚩)
+
+[Style guide](#style-guide-👗)
+
+[Recognision model](#recognition-model-🧐)
+
+[Any questions?](#any-questions-❓)
+
+
+## Getting started 👶
+
+For an introduction, please read [`README`](README.md) file. You can also find vignettes [here](vignettes/). Most likely you would find a "Getting started" article. Please also see a package manual in form of `pkgdown` website.
+## Code of Conduct
+
+This project is governed by [Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to follow the rules outlined there.
+
+## License
+
+All of your code changes would be under the same [license](LICENSE) that covers this project.
+
+## How to report a bug 🐛 ?
+
+We use GitHub to track issues, feature requests as well as bugs. Before opening a new issue please double-check if it's already reported (but don't worry if  to be a duplicate - we will manage). If already there - up-vote ⬆️. We will have a look at it faster!
+
+We kindly ask you to write a good issue with a minimal and reproducible example that showoff the problem. It's would be also great to know your local environment (R session info) as well. You will find our guides in the template while opening a bug ticket. This would speed up the fixing process a lot! 📈.
+
+## How to request the change 💡 ?
+
+In case you find a missing feature or you have an idea how to enhance existing functionalities - please let us know by creating a new issue. Before opening a new issue please double-check if it's already reported (but don't worry if occurs to be a duplicate - we will manage). If already there - up-vote ⬆️. We will have a look at it faster!
+
+Please elaborate on "why?" - what's the context, what's the benefit and for whom it is. You will find our guides in the template while opening a feature request ticket. This would help us prioritize and submit meaningful changes.
+
+## How to submit a change 🚩 ?
+
+### GitHub Flow
+
+We are following [GitHub Flow](https://docs.github.com/en/get-started/quickstart/github-flow) to collaborate in this repository. In order to submit the changes please do the following:
+
+1. Open a branch
+
+    Please see branching convention below. You can open a branch on remote as well as a fork.
+
+1. Make changes.
+
+     Make sure your code is passing all the checks, it is well documented and tested so as not to decrease existing test coverage 💪.
+
+1. Create a PR(s)
+
+    Please link an issue and make a good change description. Include development assumptions when necessary. Give all the details for a reviewer to efficiently check your changes.
+
+1. Address review comments
+
+    Please apply changes where necessary. If you disagree with a reviewer - please explain why.
+
+1. Merge your PRs and delete a branch
```

Comment:
Do you we want to give control to contributors to do the merge - feels like it's something we should do?

Reply:
once approved by any of the codeowners it should be ok